terralab/commands/pipeline_runs_commands.py

```diff
@@ -113,22 +118,43 @@ def details(job_id: str) -> None:
     LOGGER.info(
         indented(f"Pipeline Version: {response.pipeline_run_report.pipeline_version}")
     )
+    LOGGER.info(indented(f"Description: {response.job_report.description}"))
+
+    LOGGER.info(indented("Inputs:"))
+    for input_def in pipeline_response.inputs:
+        input_name = input_def.name
+        input_value = response.pipeline_run_report.user_inputs.get(
+            input_name, f"{input_def.default_value} (default)"
+        )
+        LOGGER.info(indented(f"{input_name}: {input_value}", n_spaces=4))
+
+    if response.pipeline_run_report.input_size:
+        LOGGER.info(
+            indented(
+                f"Input size: {response.pipeline_run_report.input_size} {response.pipeline_run_report.input_size_units}"
+            )
+        )
+
     LOGGER.info(
-        indented(f"Submitted: {format_timestamp(response.job_report.submitted, timestamp_format)}")
+        indented(
```

Comment:
any idewa what caused teh formatting change here?

Reply:
i didn't check, but i would guess a pylint update like you said